container-registry/container-analysis/src/main/java/com/example/containeranalysis/CreateNote.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.containeranalysis;
+
+// [START containeranalysis_create_note]
+import com.google.cloud.devtools.containeranalysis.v1beta1.GrafeasV1Beta1Client;
+import com.google.containeranalysis.v1beta1.ProjectName;
+import io.grafeas.v1beta1.Note;
+import io.grafeas.v1beta1.vulnerability.Vulnerability;
+import java.io.IOException;
+import java.lang.InterruptedException;
+
+
+public class CreateNote {
+
+  // Creates and returns a new Note
+  public static Note createNote(String noteId, String projectId)
+      throws IOException, InterruptedException {
+    // String noteId = "my-note";
+    // String projectId = "my-project-id";
+    final String projectName = ProjectName.format(projectId);
+
+    Note.Builder noteBuilder = Note.newBuilder();
+    Vulnerability.Builder vulBuilder = Vulnerability.newBuilder();
+    // Details about the your vulnerability can be added here
+    // Example: vulBuilder.setSeverity(Severity.CRITICAL);
```

Comment:
These don't seem to be right somehow.

Reply:
what doesn't seem right? The code in the comment, or the existence of the comment itself?

Reply:
Look at the line below it, it doesn't seem to match.

Reply:
The idea that I was trying to put across is that you can set instance-specific vulnerability details metadata here, but this sample keeps the "details" field empty to be more generic.

I'm going to try to improve this to be more clear

Reply:
Ok, I updated the function to add some sensible defaults. I have slightly mixed feeling about this, because I'm making the function less generic by adding these implementation details, but I'm also making the sample more useful by showing real-world usage. Let me know if you agree it's a good tradeoff